pkg/aws/policies.go

```diff
@@ -210,6 +213,52 @@ func (c *awsClient) AttachRolePolicy(roleName string, policyARN string) error {
 	return nil
 }
 
+func (c *awsClient) FindPolicyARN(operator Operator, version string) (string, error) {
+	listPoliciesOutput, err := c.iamClient.ListPolicies(&iam.ListPoliciesInput{
+		Scope: aws.String(iam.PolicyScopeTypeLocal),
+	})
+	if err != nil {
+		return "", err
+	}
+	for _, policy := range listPoliciesOutput.Policies {
+		listPolicyTagsOutput, err := c.iamClient.ListPolicyTags(&iam.ListPolicyTagsInput{
+			PolicyArn: policy.Arn,
+		})
+		if err != nil {
+			return "", err
+		}
+		skip := false
+		isTagged := false
+		for _, tag := range listPolicyTagsOutput.Tags {
+			tagValue := aws.StringValue(tag.Value)
+			switch aws.StringValue(tag.Key) {
+			case "operator_namespace":
+				isTagged = true
+				if tagValue != operator.Namespace {
+					skip = true
+					break
+				}
+			case "operator_name":
+				isTagged = true
+				if tagValue != operator.Name {
+					skip = true
+					break
+				}
+			case tags.OpenShiftVersion:
+				isTagged = true
+				if tagValue != version {
+					skip = true
+					break
+				}
+			}
+		}
+		if isTagged && !skip {
```

Comment:
I dont know if I am misreading the logic above, but what happens if there are multiple policies with the same tag `Key(s)`? 
Would the loop above always return `skip` as true, even if one policy is valid?

Reply:
No, here I want to make sure that this particular policy has tags, and that _all_ three tags (namespace, name, version) match. If any of those tags do not match, I skip it and the first `for` loop moves on to the next policy. As soon as I find a policy that has all 3 tags matching, I use it

Reply:
Perfect, thanks for the explanation.